test/syntax-highlighter.test.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * @jest-environment jsdom
+ */
+
+const syntaxHighlighter = require('../src/backend/utils/syntax-highlighter');
+
+/**
+ * syntaxHighlighter() will markup code so it can be styled as code with CSS
+ */
+describe('syntax-highlighter tests', () => {
```

Comment:
should we add a test here for it not to highlight a non-registered language?

Reply:
What would you like it to do?  Leave it as is?  I could probably check the return value of `language` and if it's not defined, just return the raw HTML again?

Or we could let it try to do the syntax highlighting, knowing it will get some cases wrong.  It might be easier to fix this when we start seeing it make mistakes, and have ready-made test cases.  Or do you have one in mind?

Reply:
> Leave it as is? I could probably check the return value of language and if it's not defined, just return the raw HTML again?
I was thinking something along the lines of this

The second test sounds great too to be honest. But as you said, that's something we can add later on when find issues with the highlighting.